test/base/wrangler.toml

```diff
@@ -0,0 +1,11 @@
+name = "wrangler-action-test"
+main = "./index.ts"
+compatibility_date = "2023-07-07"
+workers_dev = true
+
+[site]
+bucket = "./public"
+
+# [vars]
+# SECRET1 = "value1"
+# SECRET2 = "value2"
```

Comment:
```suggestion
```

Reply:
Generally suggested to have these commented out like this, we even had that in our Cloudflare Docs at one point.